modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/tests/src/Kernel/AnalyticsRulesSubscriberTest.php

```diff
@@ -0,0 +1,478 @@
+            '/nuevas' => 'news overview (default language alias)',
+            // The rule definition that matches the current path to the system
+            // path has a higher priority and takes precedence over the rule for
+            // the default language alias.
+            '/taxonomy/term/344' => 'overview of antarctican news (current path)',
+            '/news/antarctica' => 'overview of antarctican news (default language alias)',
+            '/es/nuevas/antartida' => 'overview of antarctican news (default language alias)',
+            // This currently matches on the system path instead of on the
+            // default language alias since there is no alias defined in English
+            // (which is the default language).
+            // @todo Update this one OPENEUROPA-1637 is fixed.
+            //   Once this is fixed this is expected to match on the default
+            //   language alias since this rule has a higher priority than the
+            //   system path rule. This will require a dependency on the
+            //   Pathauto module.
+            // @see https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1637
+            '/articles_page' => 'overview of articles (current path)',
+            // This is expected to match on the overview of articles in the
+            // default language alias but this is currently not working because
+            // the alias in English doesn't exist.
+            // @todo Update this once OPENEUROPA-1637 is fixed.
+            // @see https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1637
+            '/es/articulos' => '',
+          ],
+          'es' => [
+            // Since an alias in Spanish exists for the news overview page, this
+            // will match for all paths, including the system path and
+            // translations.
+            '/news_overview_page' => 'news overview (default language alias)',
+            '/news' => 'news overview (default language alias)',
+            '/nuevas' => 'news overview (default language alias)',
+            // The rule definition that matches the current path to the system
+            // path has a higher priority and takes precedence over the rule for
+            // the default language alias.
+            '/taxonomy/term/344' => 'overview of antarctican news (current path)',
+            '/news/antarctica' => 'overview of antarctican news (default language alias)',
+            '/es/nuevas/antartida' => 'overview of antarctican news (default language alias)',
+            // This doesn't match on the system path but on the default language
+            // alias since this rule has a higher priority.
+            '/articles_page' => 'overview of articles (current path)',
+            // There is no rule to define the default language alias when the
+            // site language is set to Spanish, but it should still be possible
+            // to match this using a rule that resolves the system path.
```

Comment:
My idea is that since this is an open source library it is always possible that the community can develop this feature even if the PO never prioritizes this.

I think being able to resolve system paths is very important. Currently we only support aliases, but aliases can change at will, invalidating any rules that were written. Using system paths and routes is much more resilient.